js/pulse.js

```diff
@@ -147,14 +147,14 @@
         const pos = parseInt(position);
         const insertionPointFilter = [];
 
-        if (pos & PREROLL) {
+        if (pos && PREROLL) {
```

Comment:
`&` and `&&` are very different, please revert and disable the rule for this block

Reply:
fixed within another commit